tgui/packages/tgui/interfaces/Workshop.js

```diff
@@ -118,7 +118,7 @@ const WorkshopSearch = (_properties) => {
       <Stack width="100%">
         <Stack.Item grow="1" mr="0.5rem">
           <Input
-            placeholder="Search by item name.."
+            placeholder="Введи название предмета для поиска.."
```

Comment:
```suggestion
            placeholder="Введите название предмета для поиска.."
```
Можно даже слово предмет убрать.